pkg/asset/installconfig/ovirt/validators.go

```diff
@@ -0,0 +1,77 @@
+package ovirt
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+
+	ovirtsdk "github.com/ovirt/go-ovirt"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/AlecAivazis/survey.v1"
+
+	"github.com/openshift/installer/pkg/types/ovirt"
+)
+
+// Authenticated takes an ovirt platform and validates
+// its connection to the API by establishing
+// the connection and authenticating successfully.
+// The API connection is closed in the end and must leak
+// or be reused in any way.
+func Authenticated(p *ovirt.Platform) survey.Validator {
```

Comment:
make this private as this is only used by the Platform function in this package.

Reply:
done